package.json

```diff
@@ -55,8 +55,9 @@
 		"@typescript-eslint/eslint-plugin": "^7.8.0",
 		"@typescript-eslint/parser": "^7.8.0",
 		"autoprefixer": "^10.4.19",
-		"babylonjs": "^7.25.1",
-		"babylonjs-loaders": "^7.25.1",
+		"babylonjs": "^7.42.0",
+		"babylonjs-loaders": "^7.42.0",
+		"babylonjs-viewer": "^7.42.0",
```

Comment:
If only `babylonjs-viewer` is needed as a dependency, we can remove the others.

Reply:
These are still the umd packages, so presumably they should be switched to the esm packages like in the other package.json, e.g. `@babylonjs/viewer`. `@babylonjs/viewer` also has peer dependencies on `@babylonjs/core` and `@babylonjs/loaders`, so they probably don't need to be listed explicitly.

Reply:
Assuming there isn't other code in the repo with a dependency on the UMD packages.

Reply:
I didn't find code relying on UMD code. I've removed their dependencies.